src/main/content/_includes/site-translation.html

```diff
@@ -0,0 +1,53 @@
+<script type="text/javascript"
+  src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+<script>
+  function googleTranslateElementInit() {
+    new google.translate.TranslateElement({ pageLanguage: 'en', layout: google.translate.TranslateElement.FloatPosition.TOP_LEFT }, 'google_translate_element');
+  }
+  let currentLanguage = `{{page.language}}`
+  let selectedLanguage = navigator.languages && navigator.languages[0] || navigator.language || navigator.userLanguage;
+  if (currentLanguage !== selectedLanguage) {
```

Comment:
The page language will always be `en` so no need for this `if` statement either. The swtich below will handle the cases where the selectedLanguage isn't `en`